modules/ROOT/pages/traversal-framework/bidirectional_traversal_framework.adoc

```diff
@@ -0,0 +1,81 @@
+:description: The Neo4j Bidirectional Traversal Framework Java API.
+
+:org-neo4j-graphdb-bidirectional-traversal-description: {neo4j-javadocs-base-uri}/org/neo4j/graphdb/Transaction.html#bidirectionalTraversalDescription()
+
+[[Bidirectional-Traversal-Framework]]
+= The Bidirectional Traversal Framework
+
+A bidirectional traversal consists of two traversals starting from different nodes, returning the paths where both traversals collide.
```

Comment:
Is there somewhere an image to show this? I think it could be useful to exemplify more visually what that means.

Reply:
Even better, an example to go with the picture.